### PR TITLE
Updated the TestFile.cpp to understand the github code review process

### PR DESCRIPTION
**Pull request by rijochacko**
Please Review the changes

### CODE REVIEW

File: TestFile.cpp [hunk 1]
```diff
@@ -7,9 +7,9 @@ int main()
   cout << "Enter a no" << endl;
   cin >> i;
   
-  if(0 == i)
+  if((0 == i)  || (1 == i))
```
Comment (vipilpalisakkottu) [inline TestFile.cpp]: Update the if condion like if(i < 100)

Comment (vipilpalisakkottu) [reply TestFile.cpp:10]: I will do it

Comment (rijochacko) [reply TestFile.cpp]: I will update it

**Review by rijochacko — COMMENTED** [review]
updated

File: TestFile.cpp (hunk 1 above)
Comment (rijochacko) [reply TestFile.cpp]: Updated

**Review by vipilpalisakkottu — APPROVED** [review]

**Pull request merged**